lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Lint
+      # This lint sees if there is a mismatch between the number of
+      # expected parameters for format and what is actually passed.
```

Comment:
This class comment should also include some examples of the kind of code that would trigger the cop.